lib/rubocop/cop/style/negated_if.rb

```diff
@@ -4,14 +4,38 @@ module RuboCop
   module Cop
     module Style
       # Checks for uses of if with a negated condition. Only ifs
-      # without else are considered.
+      # without else are considered. There are three different styles:
+      #
+      # both - enforces `unless` for `prefix` and `postfix` conditionals
```

Comment:
:+1: How's that? There seems to be some inconsistency in the formatting between cops—I went with what I think looks good, but let me know which cop I should mimic if this isn't right.